oslo/data/src/collators.py

```diff
@@ -0,0 +1,39 @@
+import random
+from typing import Any, Dict, List, Optional
+
+from transformers import DataCollatorForLanguageModeling
+from transformers.data.data_collator import _torch_collate_batch
+from transformers.tokenization_utils_base import PreTrainedTokenizerBase
+
+
+def tolist(x):
+    if isinstance(x, list):
+        return x
+    elif hasattr(x, "numpy"): # Checks for TF tensors without needing the import
+        x = x.numpy()
+    return x.tolist()
+
+
+class DataCollatorForGpt2:
```

Comment:
Isn't `DataCollatorForGPT2` better?

Reply:
I think we can split collator source file and move into each model directory.
as you see the `oslo.transforers.models`, there are `gpt2`directory.
how about renaming `collator_gpt2.py` and moving to this directory?